styles/styles.css

```diff
@@ -0,0 +1,168 @@
+@import url('https://fonts.googleapis.com/css?family=Noto+Sans' rel="stylesheet");
+
+body {
+  font-family: 'Noto Sans', sans-serif;
+  margin: 5px;
+}
+/* 
+#heading {
```

Comment:
Was curious about these items that are commented out but don't have any comments around them. Is this old code you didn't need anymore?